Address issue #43 (no album cover causes crash)

narutoshocker opened the pull request:
Issue:
- some songs don't have an album cover. So when you try and download them they crash the entire program with an unsuccessful download

Solution:
- modified the code so that in the event the album image doesn't exist we return an empty string.

Test case:
- tested (https://open.spotify.com/track/3iINfVB6kucFrmMRWdLMWS?si=93a9f68c06ea40bd) which is known to cause the crash. Song downloaded with no problems

narutoshocker committed:
Address issue #43 (no album cover causes crash)

Issue:
- some songs don't have an album cover. So when you try and download them they crash the entire program with an unsuccessful download

Solution:
- modified the code so that in the event the album image doesn't exist we return an empty string.

Test case:
- tested (https://open.spotify.com/track/3iINfVB6kucFrmMRWdLMWS?si=93a9f68c06ea40bd) which is known to cause the crash. Song downloaded with no problems

L0um15 commented:
Cool thanks. Can you look at line 63? Could you add a check there and i will merge it immediately. From there i will work on adding more checks and reducing duplicates in code.

narutoshocker committed:
Update SpotifyService.cs

missed line #63. 
made the same change as located in the other Album Picture sections.

narutoshocker commented:
went ahead and made that change as well @L0um15

L0um15 commented:
Thanks!

L0um15 commented:
Merged into dev branch
